src/components/Signin/index.jsx

```diff
@@ -0,0 +1,121 @@
+import { useNavigate } from "react-router-dom"
+import { useEffect, useState } from "react"
+import "./styles.css"
+
+const SingIn = () => {
+  const [password, setPassword] = useState('')
+  const [error, setError] = useState(false)
+  const [validation, setValidation] = useState(false)
+  const [email, setEmail] = useState('')
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    setError(false)
+  }, [email, password])
+
+  const handleSubmit = () => {
+    if(!email || !password ) {
+      setValidation(true)
+      return;
+    }
+    setValidation(false)
+    const user = {
+      correo: "j@text.com",
+      contrasena: "1234"
+    }
+
+    if(email === user.correo && password === user.contrasena) {
+      setError(false)
+      localStorage.setItem("token", "asdasdsdsdgsd")
+      alert("inicio seccion exitosamente")
+      navigate('/dashboard')
+    } else {
+      setError(true)
+      return
+    }
+
+  //   try {
+  //     const { data } = await axios({
+  //       method: 'GET',
+  //       baseURL: process.env,
+  //       url: '/',
+  //       headers: {
+  //         Authorization: `Bearer ${token}`,
+  //       }
+  //     })
+  //     this.setState({
+  //       message: data,
+  //     })
```

Comment:
Este código está comentado, no hay problema, pero cuidado con esas 3 lineas porque esto es un componente basado en funcion, no en clase, por ende el this.setState no tiene cabida acá. si desea guardar la data puede guardarla en un estado data como el resto de estados.